Resources/Prototypes/Datasets/Names/death_commando.yml

```diff
@@ -62,3 +60,5 @@
   - Thick McRunfast
   - Touch Rustrod
   - Trunk Slamchest
+  - Klon Klover
+  - Harry Marshal
```

Comment:
i don't know what the maintainer team policy is for little easter eggs for individual characters that are "famous" on the servers, but I consider it to be kind of bad taste. Especially since these people still actively play the game with these character names.

Reply:
Will change it then no prob